scripts.js

```diff
@@ -84,3 +89,31 @@ function generateNewPalette() {
 		}
 	}
 }
+
+function addMiniPalettesHTML(){
+	var newInnerHTML = `
+	<section class="sidebar__new-instance">
+	<section class="mini__palette"></section>
+	<section class="mini__palette"></section>
+	<section class="mini__palette"></section>
+	<section class="mini__palette"></section>
+	<section class="mini__palette"></section>
+	<section class="mini__trashcan"></section>
+  </section>`
+
+  sidebar.innerHTML += newInnerHTML;
+  for (var i = 0; i < 5; i++) {
+	  
```

Comment:
Looks good! Lets get it to add to multiple instances of mini palettes in next iteration